package.json

```diff
@@ -16,9 +16,7 @@
     "format": "prettier --write",
     "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .",
     "lint-css": "stylelint \"src/**/*.css\"",
-    "link-check": "./scripts/link-check-git-all.sh",
-    "link-check-diff": "./scripts/link-check-git-diff.sh",
-    "link-check-exclude": "./scripts/exclude-links-check.sh"
+    "link-check": "LINK_CHECK_EXCLUSION_FILE='./scripts/exclude-links.txt' LINK_CHECK_FILE_FILTER='content/*' LINK_CHECK_ROOT_URL='https://dvc.org' yarn run link-check-git-diff"
```

Comment:
should we use localhost for the ROOT URL here?

Reply:
That would only be if we could guarantee the server was up during every run.

Though a separate invocation that does that would be a great feature for offline testing!